src/components/AddBook.js

```diff
@@ -1,19 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/features/book/bookSlice';
 import '../styles/addBook.css';
 
-const AddBook = () => (
-  <div className="add-book-container">
-    <h1 className="add-book-header">Add new book</h1>
-    <form action="#">
-      <input type="text" placeholder="Book title" className="add-book-input" required />
-      <select name="new-author" id="new-author">
-        <option value="author-1">Author 1</option>
-        <option value="author-2">Author 2</option>
-        <option value="author-3">Author 3</option>
-      </select>
-      <button type="submit">ADD BOOK</button>
-    </form>
-  </div>
-);
+const AddBook = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+
+  const handleBookAdd = () => {
+    dispatch(addBook({
+      item_id: 1,
+      title,
+      author,
+      category: 'Fiction',
+    }));
+    setTitle('');
+    setAuthor('');
+  };
```

Comment:
- [ ] After adding a book, the page is getting refreshed, so the added book is not showing on the screen because the state is reset after the page refresh. Please make sure to fix this issue and the page should not get refreshed after adding a book.

Reply:
Thanks for flagging that, @thecodechaser